src/SIL.XForge.Scripture/Services/SFProjectService.cs

```diff
@@ -487,12 +485,23 @@ public async Task<string> GetLinkSharingKeyAsync(string curUserId, string projec
 
```

Comment:
In the line above this, we return null. Unless I misunderstand the frontend code, the frontend is not prepared to deal with a null value. Should we throw an exception instead of returning null, or have I missed something?